java/integTest/src/test/java/glide/TestUtilities.java

```diff
@@ -14,11 +14,12 @@ public static BaseTransaction transactionTest(BaseTransaction baseTransaction) {
         baseTransaction.set(key1, "bar");
         baseTransaction.set(key2, "baz", SetOptions.builder().returnOldValue(true).build());
         baseTransaction.customCommand("MGET", key1, key2);
+        baseTransaction.del(new String[] {key1});
 
         return baseTransaction;
     }
 
     public static Object[] transactionTestResult() {
-        return new Object[] {"OK", null, new String[] {"bar", "baz"}};
+        return new Object[] {"OK", null, new String[] {"bar", "baz"}, 1L};
```

Comment:
Can we replace "bar" with something else?

Reply:
That's what they have in their tests but I also recommend that we replace it with static value variables as we are going to implement multiple commands here.  (have an upstream PR for it) https://github.com/aws/glide-for-redis/pull/955/files#diff-2e2a26bd6d68b97dc3ec663ffa8f00287190a7107235b4caacf547ddd606c419R10

Reply:
Sorry - I just copied from their tests without thinking.  
I switched to apples/oranges in the examples app, and we can do that here too.